src/datetime/timeAgo.ts

```diff
@@ -137,7 +105,9 @@ export const timeAgo = (
 		// Simple date - "9 Nov 2019"
 		return [
 			then.getDate(),
-			verbose ? longMonth(then.getMonth()) : shortMonth(then.getMonth()),
+			verbose
+				? then.toLocaleString('default', { month: 'long' })
+				: then.toLocaleString('default', { month: 'short' }),
```

Comment:
@oliverlloyd how do you feel about these? guardian is at 98.99% support (greater than es module support). are they going to have wired side effects?

Reply:
Given that the other functions in this file return English strings, and the style guide is British English, is it possible we want to use `'en-GB'` here?

```suggestion
				? then.toLocaleString('en-GB', { month: 'long' })
				: then.toLocaleString('en-GB', { month: 'short' }),
```

I think using `'default'` will use the browser/system's default locale, and will therefore give the month in the language for that locale? For example, in French if the system is set to French. You can test this with the following snippet of code:

```js
const d = new Date();

d.toLocaleString('default', { month: 'long' });
```

in Node by setting the `LANG` environment variable:

```sh
LANG=fr_FR node
```

or in Safari by changing the macOS system language to e.g. French, restarting your browser, and then pulling up a console.

Reply:
I would be tempted to move this to a separate PR, as it’s possibly a breaking change, or at least most definitly a feature.

My suggestion for the existing code would be:

```ts
const longMonth = (month: number): string =>
	([
		'January',
		'February',
		'March',
		'April',
		'May',
		'June',
		'July',
		'August',
		'September',
		'October',
		'November',
		'December',
	] as const)[month] ?? "Unknown";
	
const shortMonth = (month: number) => longMonth(month).slice(0,3);
```

Reply:
this is a great point – `'en-GB'` should resolve that though?

Reply:
I think `then.toLocaleString('en-GB', { month: 'short' })` best respects the design guide

Reply:
I thought that [`toLocaleString` was not well supported in browsers](https://caniuse.com/mdn-javascript_builtins_date_tolocalestring), but it seems like that’s not the case, there’s only missing data for about 3% of browsers.

Reply:
Support is available for everything that supports es modules afaict?